src/main/java/org/jenkinsci/plugins/github/config/GitHubTokenCredentialsCreator.java

```diff
@@ -141,13 +140,17 @@ public FormValidation doCreateTokenByCredentials(
 
         GHAuthorization token;
 
+        if (null == creds) {
+            return FormValidation.error("Can't create GH token - credentials are null.");
+        }
+
         try {
             token = createToken(
-                    notNull(creds, "Why selected creds is null?").getUsername(),
-                    creds.getPassword().getPlainText(),
+                    creds.getUsername(),
+                    Secret.toString(creds.getPassword()),
                     defaultIfBlank(apiUrl, GITHUB_URL)
             );
-        } catch (IOException e) {
+        } catch (Exception e) {
```

Comment:
why?

Reply:
Wooops, this was unintended, this block of code was giving me hell for a while, and at one point I was trying to catch `IOException` and the `NPE` from `notNull`, I will revert this.